x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -36432,7 +36431,6 @@
     "xpack.transform.transformHealth.yellowLabel": "Dégradé",
     "xpack.transform.transformList.alertingRules.screenReaderDescription": "Cette colonne affiche une icône lorsqu'il existe des règles d'alerte associées à une transformation",
     "xpack.transform.transformList.cloneActionNameText": "Cloner",
-    "xpack.transform.transformList.completeBatchTransformBulkActionToolTip": "Une ou plusieurs transformations sont des transformations par lots terminées et ne peuvent pas être redémarrées.",
```

Comment:
Since we are just renaming ids, should we update the ids for these translations files as well to avoid removing them completely?

Reply:
Fixed in ca7197e131c.